### PR TITLE
Add default fallback if no customer name

### LINKED ISSUE #28653: Error on customer service thread page - PHP 8.1
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

With a fresh install, an error appears on the view thread message page in customer service tab in BO.

A first message is send, no error. Screenshot below:

Capture d’écran 2022-06-01 à 11 30 08

A second message is send error appears. Screenshot below:

- With long text

Capture d’écran 2022-06-01 à 11 30 34

- With short text

Capture d’écran 2022-06-01 à 14 39 26

Video below to see the behavior:

https://user-images.githubusercontent.com/2854834/171403214-ff08d56a-1f8b-45b5-a409-2d2eb484789b.mp4

With or without attachment / Multistore activated or deactivated / Customer service or webmaster subject => error appears with the 2nd message

Tested on develop and 1.7.8.x branch with 7.2 php version and no error

### Expected behavior

No error appears on this page

### Steps to reproduce

1. Go to FO > "Contact us" page on header or footer
2. Be a guest customer and fill the form with your email and message
3. Click on send button
4. Go to BO > Customer service > Customer service
5. Click on the view button on the thread 
6. Go to FO > refresh the "contact us" page
7. Fill the form again with the same email and other message
8. Click on send button
9. Go to BO > refresh the thread page
10. See error

### PrestaShop version(s) where the bug happened

develop

### PHP version(s) where the bug happened

8.1

### If your bug is related to a module, specify its name and its version

_No response_

### PR DESCRIPTION
**Pull request by atomiix**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Since PHP 8.1, a deprecated notice is raised if we try to display non-existing customer name. This PR aims to fix it by adding an empty string as a fallback.
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #28653
| Related PRs       | 
| How to test?      | Please see #28653
| Possible impacts? |

### CODE REVIEW

File: admin-dev/themes/default/template/controllers/customer_threads/helpers/view/message.tpl [hunk 1]
```diff
@@ -46,7 +46,7 @@
 			<h4 class="message-item-heading">
 				<i class="icon-mail-reply text-muted"></i>
 					{if $type == 'customer'}
-						{$message.customer_name|escape:'html':'UTF-8'}
+						{$message.customer_name|default|escape:'html':'UTF-8'}
```
Comment (matthieu-rolland) [inline admin-dev/themes/default/template/controllers/customer_threads/helpers/view/message.tpl:49]: I'm not sure I understand the behavior of `default` smarty modifier when it's not given a string parameter, it's empty? 

Then if `message.customer_name` is false you want an empty string?  (just to be sure I understand well)

**Review by matthieu-rolland — COMMENTED** [review]
one question...

File: admin-dev/themes/default/template/controllers/customer_threads/helpers/view/message.tpl (hunk 1 above)
Comment (matthieu-rolland) [reply admin-dev/themes/default/template/controllers/customer_threads/helpers/view/message.tpl:49]: ah I read your PR description too fast, sometimes the key doesn't even exist, ok

**Review by matthieu-rolland — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Comment by AureRita** [comment]
Hi @atomiix ,

Thank you for your PR, I tested it and I no longer have the red notice for the message, as we can see on this video : 

https://user-images.githubusercontent.com/100699445/172618303-37cf6e0c-eae5-47f5-9003-43467fd8cdc5.mp4

but unfortunately, I currently have a deprecated message after sending a message by the FO : 
image

**Comment by atomiix** [comment]
@AureRita this message comes from the contactform module and [a PR has already been merged](https://github.com/PrestaShop/contactform/pull/61) to fix it but the module hasn't be released yet.

**Review by AureRita — APPROVED** [review]
Thank for this detail, indeed, this issue is already fixed in the module contactform 

Thank you @atomiix

**Comment by Progi1984** [comment]
Thanks @atomiix & @AureRita

**Pull request merged**